publish_json_schema/publish_json_schema.py

```diff
@@ -0,0 +1,9 @@
+from marshmallow import Schema, fields
+
+
+class PublishJSONSchema(Schema):
+    """Schema for Publish."""
+
+    id = fields.String(required=True)
+    data = fields.Raw(required=True)
```

Comment:
I think this is rather a `fields.Dict` than `fields.Raw`. Because [later on](https://github.com/ManageIQ/aiops-publisher/blob/37d24ed70202a24ad2caab696fe7d2bfd35f138b/wsgi.py#L35) we take this data and serialize it via `json.dumps`.

Reply:
sure.
I have changed it to `fields.Dict` - 4427d45